2017-03-01 13:28:10 INFO [Testcase_2002.Testcase_2002.test_1000_1] Step 1 Started.
Traceback (most recent call last):
  File "/Users/lex/Desktop/s/projecttwo/libs/product/omm/standard/ui_controls.py", line 36, in ui_element
    element = g.driver.find_element_by_id(identifier_val)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 266, in find_element_by_id
    return self.find_element(by=By.ID, value=id_)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 744, in find_element
    {'using': by, 'value': value})['value']
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 233, in execute
    self.error_handler.check_response(response)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/appium/webdriver/errorhandler.py", line 29, in check_response
    raise wde
NoSuchElementException: Message: An element could not be located on the page using the given search parameters.

Traceback (most recent call last):
  File "/Users/lex/Desktop/s/projecttwo/libs/product/omm/standard/ui_controls.py", line 107, in text_box
    if text_box_element.is_enabled():
AttributeError: 'NoneType' object has no attribute 'is_enabled'
Traceback (most recent call last):
  File "/Users/lex/Desktop/s/projecttwo/libs/product/omm/standard/ui_controls.py", line 36, in ui_element
    element = g.driver.find_element_by_id(identifier_val)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 266, in find_element_by_id
    return self.find_element(by=By.ID, value=id_)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 744, in find_element
    {'using': by, 'value': value})['value']
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/selenium/webdriver/remote/webdriver.py", line 233, in execute
    self.error_handler.check_response(response)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/appium/webdriver/errorhandler.py", line 29, in check_response
    raise wde
NoSuchElementException: Message: An element could not be located on the page using the given search parameters.

Traceback (most recent call last):
  File "/Users/lex/Desktop/s/projecttwo/libs/product/omm/standard/ui_controls.py", line 314, in button
    if element_button.is_enabled():
AttributeError: 'NoneType' object has no attribute 'is_enabled'
2017-03-01 13:28:18 CRIT [Testcase_2002.Testcase_2002.test_1000_1] Fail:OMM login is not successful - 
2017-03-01 13:28:18 CRIT [Testcase_2002.Testcase_2002.test_1000_1] Step 1 error: <b>EXPECTED RESULT:</b><br/>User should be able log in to OMM<br/><br/><b>ACTUAL RESULT:</b><br/>OMM login is not successful - <br/>
2017-03-01 13:28:18 INFO [Testcase_2002.Testcase_2002.test_1000_1] Step 1 Completed.
2017-03-01 13:28:18 INFO [Testcase_2002.Testcase_2002.test_1000_1] Step 2 Started.
